lib/utils.js

```diff
@@ -602,14 +602,21 @@ module.exports.processMethod = async function (msg, cfg, snapshot, TOKEN) {
       && (reboundErrorCodes.has(e.response.status)
         || e.message.includes('DNS lookup timeout'))
     ) {
-      emitter.logger.info('Component error: %o', e);
+      emitter.logger.info('Component error: %o', {
```

Comment:
Are we losing some visibility with this. I think adding JSON.stringify(e) might be better than selecting, but not sure what kind of noise that creates.